pipenv/utils/requirementslib.py

```diff
@@ -654,7 +655,7 @@ def unpack_url(
         would ordinarily raise HashUnsupported) are allowed.
     """
     # non-editable vcs urls
-    if link.scheme in [
+    if link.scheme in {
```

Comment:
While you are at it, can we update to use `VCS_SCHEMES` from constants?

Reply:
Of course! I've now updated the code. I thought the differences with constant might cause problems